globals/card_definitions.gd

```diff
@@ -1010,6 +1035,8 @@ func get_effect_type_text(effect, card_name_source : String = "", char_effect_pa
 				effect_str += "+" + str(effect['amount']) + "-" + str(effect['amount2']) + " Range per boost in play."
 		"rangeup_per_card_in_hand":
 			effect_str += "+" + str(effect['amount']) + "-" + str(effect['amount2']) + " Range per card in hand."
+		"speedup_per_force_spent_this_turn":
+			effect_str += "+" + str(effect['amount']) + "-" + str(effect['amount2']) + " Range per force spent this turn."
```

Comment:
this should be speed not range in the string?

Reply:
oh, no you copied it below, so this one should have the case be rangeup

Reply:
whoops, thanks